Feature/oas preview

domharrington opened the pull request:
Add preview site to replace https://swagger.readme.io/preview/aHR0cDovL3BldHN0b3JlLnN3YWdnZXIuaW8vdjIvc3dhZ2dlci5qc29u#tag__store

mjcuva commented:
Shouldn't it be using the path instead of the tag as the page title? 

![https://www.dropbox.com/s/d2tuifekl2jou0s/Screenshot%202018-09-25%2016.34.25.png?raw=1](https://www.dropbox.com/s/d2tuifekl2jou0s/Screenshot%202018-09-25%2016.34.25.png?raw=1)

domharrington commented:
I think you're right... though this also appears to be broken on here too: https://readmeio.github.io/api-explorer/?selected=https://readme.readme.io/openapi/5a95f7d15c3020002b02aace

I think this would be fixed by adding path to this line? https://github.com/readmeio/api-explorer/blob/94c3984367c492c03d27cfa62364ab688a48c457/packages/api-explorer/lib/create-docs.js#L44

domharrington commented:
Fixed the issue with the tags, now they get the path:

<img width="827" alt="screen shot 2018-09-26 at 15 46 40" src="https://user-images.githubusercontent.com/848223/46113471-6568d800-c1a3-11e8-9557-55b3f65d9c2b.png">

Also added some error handling if whatever they give us is invalid JSON etc:

<img width="822" alt="screen shot 2018-09-26 at 15 47 42" src="https://user-images.githubusercontent.com/848223/46113500-85989700-c1a3-11e8-9a05-70d2832c3229.png">

We do not validate their swagger file here like we used to, but i think this is fine for now.
